prisma/schema.prisma

```diff
@@ -16,71 +16,69 @@ model Gender {
 }
 
 model Neighborhood {
-  id                  Int                   @id @default(autoincrement())
-  adminID             Int                   @unique
-  admin               User                  @relation(fields: [adminID], references: [id], onDelete: Cascade, onUpdate: NoAction)
-  name                String                @db.VarChar(50)
-  description         String?               @db.VarChar(50)
-  location            String?               @db.VarChar(50)
-  users               NeighborhoodUsers[]
+  id                  Int                 @id @default(autoincrement())
+  admin_id            Decimal             @db.Decimal
```

Comment:
why is that Decimal instead of Int?

Reply:
In the initial SQL schema, the id was defined as being numeric, not an integer. Prisma converted numeric to decimal. I think we can safely change it to integer.